src/lib/index.js

```diff
@@ -31,6 +31,7 @@ lib.parseDate = datesModule.parseDate;
 var searchModule = require('./search');
 lib.findBin = searchModule.findBin;
 lib.sorterAsc = searchModule.sorterAsc;
+lib.sorterDec = searchModule.sorterDes;
```

Comment:
*Decreasing* or *Descending*?

We should keep it consistent with increasing/inc and decreasing/dec or ascending/asc and descending/des.